test/e2e/framework/test_context.go

```diff
@@ -364,6 +367,7 @@ func RegisterCommonFlags(flags *flag.FlagSet) {
 
 	flags.StringVar(&TestContext.E2EDockerConfigFile, "e2e-docker-config-file", "", "A docker credentials configuration file used which contains authorization token that can be used to pull images from certain private registries provided by the users. For more details refer https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#log-in-to-docker-hub")
 	flags.StringVar(&TestContext.KubeTestRepoList, "kube-test-repo-list", "", "A yaml file used for overriding registries for test images. Alternatively, the KUBE_TEST_REPO_LIST env variable can be set.")
+	flags.StringVar(&TestContext.KubeTestImageRepository, "kube-test-repo", "", "The image repository where the test images are hosted, like \"quay.io/openshift/community-e2e-images\". Alternatively, the KUBE_TEST_REPO env variable can be set.")
```

Comment:
Would it make sense for the default to be `os.Getenv("KUBE_TEST_REPO")`?

Reply:
> Would it make sense for the default to be `os.Getenv("KUBE_TEST_REPO")`?

Referencing `kube-test-repo-list` a peer configuration, I think not.